.github/workflows/build-example-desktop-mac.yml

```
@@ -0,0 +1,41 @@
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          cd packages/connect-examples/electron-example
+          yarn make:mac
+
+      - name: Upload macOS Artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: mac-example-desktop
+          path: packages/connect-examples/electron-example/out/*.dmg
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)